conda_forge_tick/auto_tick.xsh

```diff
@@ -446,6 +446,7 @@ def initialize_migrators(do_rebuild=False):
 
     add_arch_migrate($MIGRATORS, gx)
     add_rebuild_openssl($MIGRATORS, gx)
+    add_rebuild_successors($MIGRATORS, gx, 'fortran_compiler_stub', '7')
```

Comment:
This would make a confusing PR title with `fortran_compiler_stub`.

Reply:
Doesn't this string have to match how the bot mangles the compiler versions internally when figuring out how to migrate things?

Reply:
Yes, you'll need to subclass the rebuild class and change the PR title.

Reply:
Ahhhh ok. I understand the code correctly. Yay!